Dockerfile

```diff
@@ -13,8 +12,7 @@ ARG USE_VAULT_ARG
 # - local
 ENV USE_NGINX_PLUS=${USE_NGINX_PLUS_ARG:-true} \
     USE_VAULT=${USE_VAULT_ARG:-false} \
-    APP="unicorn -c /usr/src/app/unicorn.rb -D" \
-    CONTAINER_ENGINE=${CONTAINER_ENGINE_ARG:-kubernetes}
+    APP="unicorn -c /usr/src/app/unicorn.rb -D"
```

Comment:
We can move this to the start script so that it is consistent with the rest of the services.

The APP environment variable is not used in build script or Dockerfile